css/style.css

```diff
@@ -349,15 +328,15 @@ pre code {
 #top {
   line-height: 0;
   background: black;
-  -webkit-border-radius: 2px;

```

Comment:
Also maybe leave in this webkit prefix, but also keep your added non-prefixed change.

Reply:
The same, according to [caniuse](https://caniuse.com/border-radius), it has good support.